---
title: "Project Summary"
header-includes:
   - \setlength\parindent{24pt}\setlength{\parskip}{0.0pt plus 1.0pt}
output:
  pdf_document:
    latex_engine: xelatex
mainfont: Times New Roman
fontsize: 16
---
\large
\makeatletter
\newcommand\iraggedright{%
  \let\\\@centercr\@rightskip\@flushglue \rightskip\@rightskip
  \leftskip\z@skip}
\makeatother
\iraggedright





\begin{raggedleft}
\textbf{Overview}  
\end{raggedleft}  

  

<<<<<<< HEAD
Why does historical adaptation lead to contemporary constraints? 
=======
Why does historical adaptation lead to contemporary constraints? Low standing genetic variation in functionally relevant traits may be the key limiting factor in response to novel selection regimes. On the other hand, even if standing genetic variation is large, negative correlations between functionally relevant traits may constrain evolutionary responses. However, the genetic basis of adaptation appears more complex because the genotype to phenotype map shifts alongside environmental gradients, suggesting that even the same trait has a dynamic genomic basis when expressed under different environments, yet we understand very little about this process. Thus, significant gene by environment interaction leads to difficulty in predicting population responses to new environments, particularly when responses are non-linear. 
>>>>>>> Project_description

*Suggest slight change to: "How does historical selection on natural variation lead to contemporary constraints on growth and productivity under novel environments?"*

Low standing genetic variation in functionally relevant (*functionally relevant probably redundant; perhaps just "functional traits"?*) traits may be the key limiting factor in response to novel selection regimes. On the other hand, even if standing genetic variation is large, negative correlations between functionally relevant (*same*) traits may constrain evolutionary responses. However, the genetic basis of *some adaptive traits* appears more complex because the genotype to phenotype map shifts alongside environmental gradients, suggesting that even the same trait has a dynamic genomic basis when expressed under different enviornments, yet we understand very little about this process. (*suggest break this last one up into 2 sentences*) Thus, significant gene x environment interaction leads to difficulty in predicting *genotypic growth* responses to new environments, *which may be especially challenging* when responses are non-linear. 


Non-linear responses of a phenotype with respect to the environment is best imagined as a continuous function, known as a function-valued trait. Despite a rich literature and development of quantitative genetic theory surrounding function-valued traits, we know very little about *their* genomic *architecture*. For example, if a function-valued trait occurs as a logistic function, the exponential portion may relate to regulatory gene elements which are responsible for gene expression, while the maximum portion should be more associated with sequence variation which are related to activity. These hypotheses are yet to be explored.

*interesting predictions...are they yours, or based in theory from another reference?*

(*Here, I would focus on "poplar" sensu lato, and introduce the specific system of balsam later*)
*Poplulus* (represents) *are*(an) economically and ecologically important tree*s*(species) that *provide a powerful*(is a ripe) system to understand *the impact of historical seletion and* gene by environment interactions *on productivity*. *Populus is a DOE flagship genus for woody biomass production, and abundant natural genomic and phenotypic variation has been characterized in P. balsamifera due to its close ancestry to the model P. trichocarpa, and its more expansive range which spans substantial differences in growing season length*. (particularly due to the vast natural history and genomic resources available.) *Clonal propagation in Populus also is ideal for assessing genotypic responses in functional-valued traits across environmental gradients, and clonal selection is the principal mode used by plant breeders.* (They can be propograted clonally which makes them amenable for common garden experiments.) Current common garden *experiment*s show (that populations display) *genetically based* clines in phenology with signatures of positive selection *and gene-environment association* in genomic regions related to phenology *and growth*. However, growing season length *cues driven by photoperiod vs.*(and) temperature are *typically* confounded in these analyses because common gardens are planted along a latitudinal gradient. Therefore, **I propose to 1) identify genetic (*adaptive?*) constraints in woody biomass production (over varying) *across an environmental gradient of*) growing season lengths while controlling for (temperature) *photoperiod* and 2) uncover the genomic landscape of trade-offs associated with growing season lengths**.    \newline


\begin{raggedleft}
\textbf{Intellectual Merit}  
\end{raggedleft}   
 
This work will contribute to understanding the limits of adaptation *and productivity* (to) *under novel*(changing) environments *such as climate change*, particularly when the genetic architecture of adaptation involves integration of function-valued traits. This project will link quantitative genetic theory with the genomic underpinnings of an ecological and economically important trait: woody biomass production. Therefore, proper identification of trade-offs not only aide in predicting resiliency or susceptibility to climate change, but it also has an applied value. No single genotype is best suited for all environments and this work will inform optimal selection of cultivars for (poplar farmers) *plant breeders and growers*. \newline


\begin{raggedleft}
\textbf{Broader Impact}  
\end{raggedleft}   
(No single scientist operates on a secluded island and it is my firm belief that the process of science is a social endeavor) *--I agree completely, but you should have something more hard hitting here*. (Therefore, )I will work closely with my post-doctoral mentor, Dr. Stephen Keller, to develop *my foundations and* (a greater foundation in ecological genetics and further enhance my pre-existing skills) *in population* genomic analyses *as well as my abilities as a teach and mentor*. *A key component of this will be involving* (I will also mentor) undergraduate researchers in the plant biology department at the University of Vermont *in this research,* with the goal of leading them to (first) *co-*author*ed* publications. Burlington, Vermont is a resettlement site for refugee from Nepal, Vietnam, and Africa; therefore funding from NSF will allow me to engage with these under-represented miniorities. (*How are you thinking of doing this? As UVM students? Or otherwise engaging this community otuside of an academic setting?*) Burlington is one of my common garden sites, therefore, I will (offer the opportunity for) *engage* local middle and high school students to participate in phenotyping poplar *with the aim of fostering appreciation for the importance of genomics to sustaining plant productivity under climate change* and *providing real life experiences of* how science works. Experiencing and participating in science is absolutely critical for fostering an interest in science. In fact, my fascination with the natural world did not occur until college and one of my major goals is to open young minds to the prospect that becoming a scientist is a real profession. 

Outside of interacting with local communities, I will share my research with other scientists and poplar (farmers) *breeders*. First, I will attend conferences to dessiminate critical findings via oral presentations. All data generated from this project will be stored and shared on open data repositories such as *NCBI SRA, iPlant, and* Github*, with fully documented workflows to faciliate repeatable analyses*. Therefore, any scientist (or even non-scientist) may repeat and reproduce analyses. Second, I will reach out to (local) (poplar farmers)*the tree breeding community* to (prescribe) *transfer knowledge about identifying functional-valued traits and using genomic prediction to identify* genotypes that will *perform* optimally (grow) in (their)*a given environment*() region). To illustrate my findings, I will create an *interactive web user-interface* (e.g., a Shiny application) (which is an interaction web user-interface) that will *facilitate*(display)*transfer and interpretation of* scientific results in a digestable manner. 



