---
title: "Project Description"
header-includes:
   - \setlength\parindent{24pt}\setlength{\parskip}{0.0pt plus 1.0pt}
output: 
  pdf_document:
    latex_engine: xelatex
mainfont: Times New Roman
fontsize: 16
---

\large
\makeatletter
\newcommand\iraggedright{%
  \let\\\@centercr\@rightskip\@flushglue \rightskip\@rightskip
  \leftskip\z@skip}
\makeatother
\iraggedright


\begin{raggedleft}
\textbf{Introduction} 
\end{raggedleft}   

Adaptation to historical climate conditions often lead to constraints in current conditions (Etterson & Shaw 2001; Weins 2004), particularly if populations are exposed to novel environments such as those brought about by climate change (IPCC 2007). **The degree to which organisms can alter their functional traits, known as phenotypic plasticity, across shifting climate gradients dictates whether they are resilient or susceptible, which is especially important for long-lived species** that experience significant environmental variation within their lifetimes (Gilchrist 1995). Typically, phenotypic plasticity is reduced during climate adaptation such that physiological mechanisms that were once induced become fixed (Waddington 1953; Pigliucci et al. 2006), resulting in trade-offs between historical and contemporary environments that may indicate susceptibility to climate change. However, changes in the environment may interact in unique ways with genotypes such that trade-offs lessen or may even completely reversed (Messina & Fry 2003), resulting in resiliency to environmental perturbations. Yet, **we know very little about the complex dynamics of trade-offs** because many experimental approaches (Wu et al. 1992; Wu & Stettler 1997; Zalesney et al. 2005) do not include environmental manipulations that are continuous in nature which may severely underestimate the genotype by environmental influences (Stinchcombe et al. 2012). 

Genotype by environmental interaction that produce phenotypes (Yamada 1962) often manifest in non-linear responses that can be represented as mathmetical expression, known as a *function-valued trait* (Gomulkiewicz & Kirkpatrick 1992; Kingsolver et al. 2001). **Visualizing a trait as such offers insights on the nature of trade-offs**, for example, genotypes may perform highest at specific windows of the environmental gradient (*short-long trade-off*) or perform higher over wider or shorter windows (*generalist-specialist trade-off*; Kingsolver et al. 2015). Unfortunately, we have very little understanding for how complex environmental axes modulate trade-offs of function-valued traits, both at the phenotypic and genomic levels, which critically impairs predictions of species responses to climate change (Gienapp et al. 2008; Fitzpatrick & Keller 2014).

Shifting climate may potentially negatively impact **Poplar, a DOE flagship genus with long-lived species** that provide key ecosystem services (Schimel et al. 1998) and are utilized as an alternative energy source (Stanton et al. 2010). Poplar is an exemplary model to understand how historical selection regimes have shaped populations can result in either resiliency or susceptibility to future climate change. In particular, populations of balsam poplar (*Populus basamifera*) display strong signatures of historical selection on productivity (Kellet et al. 2011) and genetic based clines in phenology in natural populations (Keller et al. 2012) and under common gardens (Keller et al., unpublished). **However, growing season length (GSL) cues driven by photoperiod or temperature are confounded and future experimental work needs to disentangle both forces **(Keller et al., unpublished? ). Furthermore, since climate change will bring about more complex shifts such as more variable precipitation regimes (Mueller & Seneviratne 2012), whether optimal hydration status can rescue or reinforce trade-offs in GSL is currently unknown, especially since soil moisture is a critical parameter of growth for balsam poplar (LarchevÃªque et al. 2011). Luckily, balsam poplar readily propogate clonally and in fact, clonal selection is the principal mode used by plant breeders (Stanton et al. 2010). Therefore, balsam poplar are amenable to quantitative genetics and genomics techniques that will improve predictions to future climate shifts (Stanton et al. 2010; Fitzpatrick & Keller 2014). \newline

\begin{raggedleft}
\textbf{Research Objectives}
\end{raggedleft}   
**My overall goal is to determine the quantitative genetic architecture of trade offs in growth, imagined as a function-valued trait, across 2 continuous environmental axes: GSL and soil moisture**. Genetic constraints will be determined with two interdisciplinary approaches: **1)** Assess the shifts in the genetic architecture in growth performance both at the phenotypic and genomic levels and **2)** Quantify any changes in the effects of previously identified quantitative loci associated with shifts in the genetic architecture and predict performance broadly to natural populations.


![](Figures_related/20160819_sampling_map_common_gardens.pdf)     

>**Figure 1. 11 sites vary in their growing season length (Julian Days) with very similar climate in New York and Vermont.** 

\begin{raggedleft}
\textit{Experimental Approach}
\end{raggedleft}  
With the involvement of **high school students, undergraduate researchers, and local botany groups (New England Wild Flower Society and Burlington Garden Club)**, we will establish 8-12 common gardens that span a GSL gradient spanning 2.5 months (Figure 1) using a quantitative genetic clonal design. In addition, we will vary soil moisture at 2 levels, high or low, at each common garden. Within each common garden and moisture level, we will randomly plant 30-50 unique genotypes with 3-5 replicates per genotypeat 1-2 meter spacing. Prior to planting, genotypes will be clonally propogated utilizing the University of Vermont's greenhouse to initially control for growth conditions and ensure enough material for replication. Genotypes will be selected from regions spanning the whole climate gradient of their native range and have pre-existing genomic resources/information.  **In total, the community will have established 1,440-6,000 poplar plants**. 

Under the common garden design, we will quantify phenotypes at the higher level (growth) and at the lower molecular level (mRNA) that drives phenotypic plasticity. We will measure mRNA with next-generation sequencing. Using traditional quantitative genetic methods, we can identify the trade-offs in function-valued growth and among all the mRNAs. Furthermore, we will assess whether these trade offs correspond to previously identified genomic regions (quantitative trait loci). 

\begin{raggedleft}
\textit{Anticipated Major Outcomes}
\end{raggedleft}  
When reared along a GSL gradient, genotypes may exhibit no constraints at all in growth (Figure 2 A,D), or two types of trade-offs in growth: shorter-longer (Figure 2 B,E), or generalist-specialist (Figure 2 C,F). Elevated soil moisture may either **lessen trade-offs** (i.e. slope in shorter-longer variation becomes less steep, Figure 3 A), **shift between trade-offs** (shorter-longer to generalist-specialist, Figure 3 B), or even **reverse trade-offs** (i.e. shorter-longer variation at one moisture level and generalist-specialist, Figure 3 C). \newline

![](Figures_related/perf_curves.pdf)    

 >**Figure 2. Predicted outcomes for growth performance (function-valued trait) along a growing season length (GSL) environmental gradient.** The top panels (**A,B,C**) illustrates different types of performance curves (growth) as a function of GSL for mock clones. The accompanying bottom panels  (**D,E,F**)illustrates the genetic correlation (loadings from multivariate analyses) between growth and GSL using quantitative genetics techniques (Kingsolver 2001; Kingsolver et al. 2015). If **overall variation in growth is not constrained** (**A**), then growth is genetically correlated among all GSL conditions (**B**). However, if there is a **trade-offs between shorter-longer GSL** (**C**), then there will be opposing correlational structuring of performance among growing length condtions (**D**). Lastly, if there is **generalist-specialist trade-offs** (**E**), then performance at intermediate GSL are negatively correlated with extreme GSL (**F**).   
 
\begin{raggedleft}
\textit{Significance}
\end{raggedleft}
The proposed work will illuminate the nature of trade-offs for two critical environmental axes (GSL and water availability). **Determining whether trade-offs can be alleviaed or reversed has significant applied value** because poplar growers may enact management practices to maximize yield, particularly for an ecologically and economically important species within poplar, a DOE flagship genus. If we observe **overall variation** in growth, then the top cultivar operates the highest over many latitudes (Figure 2 A,D). However, if there are constraints between **shorter-longer GSL**, then only particular cultivars should be chosen at a given latitude (Figure 2 B,E). Finally, **generalist-specialist GSL** would indicate that specific latitudes should utilize a certain cultivar (Figure 2 C,F). \newline

![](Figures_related/GSL_pred_trades.pdf)   

>**Figure 3. Three critical anticipated results from proposed acitivties illustrating the interaction between GSL and soil moisture on growth**. Trade offs may be lessened (**A**), shift between them (**B**), or become reversed (**C**) under high and low soil moisture treatments. 

\begin{raggedleft}
\textbf{Training Objectives} 
\end{raggedleft}
I have **3 general training objectives**: 1) Establish a solid foundation in ecological genomics and apply these principles to a new system plant systems; 2) Acquire more statistical and bioinformatic tools to uncover complex biological patterns associated with genomic and phenotypic datasets; 3) Effectively communicate science to diverse backgrounds.    

**Goal 1**: A foundation in the natural history of balsam poplar and population genetic theory will be absolutely critical for carrying out the proposed work. To achieve this goal, I will aide in ongoing common gardens in Burlington, VT and Indian Head, Canada, which will allow me to become familiar with poplar and interact with experts in the field. I will lead journal clubs specific to population genetic papers in order to solidify my scientific purview. After all, I learn best by talking and doing science.  

**Goal 2**: Although I have worked with next-generation genomic data in my dissertation, there is much more to learn. In particular I will need to create genotype-phenotype maps, project genomic predictions, and manage large datasets. To aide in this goal, I will consult and meet with my post-doctoral advisor on a weekly basis when data are ready to be analyzed.

**Goal 3**: Because science will solve pressing issues that we face today, it is my firm belief that science should be shared to all people from different walks of life. I will develop undergraduate projects and offer one on one mentoring. Next, I will actively include local high school students in their 2 week **Year End Semester (YES) program** and  **members from the New England Wild Flower Society (http://www.newenglandwild.org/) and Burlington Garden Club (http://www.bgcvt.org/)** into our common garden in order to cultivate citizen scientists. I will communicate my findings to poplar farmers. As a member of the **Burlington Data Science Meetup (http://www.meetup.com/Burlington-Data-Scientists/)**, I will host a **Hackathon** where local data scientists from diverse professions may propose novel methods for uncovering biological patterns. \newline

\begin{raggedleft}
\textbf{Career Development}  
\end{raggedleft}   
Funding will facilitate my long term goal: **discover and uncover novel patterns in biological processes, whether as an academic, consultant, or data scientist**. Today's problems require handling big-data and synthesis between fields and I am would be thrilled to be given the opportunity to develop solutions. Funding for the proposed work will allow me create explicit links between two fields: quantitative genetics and the molecular bases of function-valued traits. To create these links, I will further develop my statistical and informatic skills to analyze large datasets generated from next-generation sequencing.    
Although the research theme is similar, the proposed work is substantially different from my dissertation. My dissertation focused on how molecular level processes explains diversity at the species-level and utilizing this information to predict ant species responses to climate change. However, species comprise of many populations, whose responses may vary themselves. Therefore, **as an NSF post-doctoral fellow, I hope to approach climate change adaptation at the population-level using a completely different system from my dissertation work (ants), such as balsam poplar**. \newline

\begin{raggedleft}
\textbf{Choice of Sponsoring Scientists}
\end{raggedleft}
In collaboration with **Dr. Stephen Keller in the plant biology department** at the University of Vermont, I will be able to explore my ideas in bridging links between quantitative genetics and the genomic basis of function-valued traits on an economically and ecologically important cottonwood tree species (balsam poplar). Trees do not move, unlike the ants I am accustomed to, and are amenable to quantitative genetic analyses because they clonally propogate. Dr. Keller has been working on balsam poplar for 10 years and has established an immense amount of genomic resources through funding from the **National Plant Genome Initiative (NPGI)**. Over the span of his career, Dr. Keller has published **27 peer-reviewed manuscripts**, utilizing diverse approaches such as field surveys, lab work, and bioinformatics to answer pressing issues that we face today. This high productivity and track record, which I strive to emulate, identified significant climate adaptation for phenology in balsam poplar with strong signatures of selection for loci relating to growth and phenology and integrated these findings into climate projection models. Therefore, **Dr. Keller has established himself as a pioneer and leader in climate change biology**.\newline

\begin{raggedleft}
\textbf{Broader Impacts and Timetable} 
\end{raggedleft}   
The proposed activities will achieve greater understanding of how species will respond to climate change on a completely different system (plants) from my dissertation work (ants) under the guidance and expertise of Dr. Stephen Keller and with the involvement of diverse citizens. Since planting thousands of poplar is an enormous task, I will utilize local high schoolers, undergraduate researchers and members from local botanical groups (New England Wild Flower Society and Burlington Garden Club,Table 1, #2, #3) to help establish common garden plots (Table 1, #1, 2017) and participate in research activities (Table 1, #1 and #2 2018-2020). I will leverage **Burlington High School's 2 week Year End Semester (YES) program (Table 1, #2, 2017-2020)** and recruit students to aide in phenotyping plants. Students from diverse backgrounds will be particularly represented because Burlington is a target site for refugees from African countries (~20), Nepal, and Vietnam and will interact with undergraduate researchers and citizen scientists. For undergraduate researchers, I will develop projects through individual mentorship (Table 1, #3, 2018-2019) and translate their findings into first author publications in open access journals (Table 1, #3 and #7 2020). Since the University of Vermont strongly supports undergraduate investigators to showcase their research, I will bring them to conferences so they can present posters and/or oral presentations (Table 1, #3 and #6, 2019-2020).

Data collection will be streamlined into transparent and reproducible workflows on Github (https://github.com/adnguyen, Table 1, #4, 2017-2020), enabling effective dessimination of proposed activities within the YES program, to other scientists, and even poplar farmers (Table 1, #2, 2020). Specifically,  **I will create an educational interactive web application (Shiny App, Table 1, #2, 2018)** to illustrate how different genotypes' growth performances can be represented by mathmatical functions. **As a member of the Burlignton Data Science Meetup group, I will also host a hackathon (Table 1, #2, 2019)** so that local data scientists are exposed to statistical approaches in biological systems. In particular, I will propose the problem of analyzing thousands of transcripts, collected in 2018 (Table 1, #5), where each transcript represents a function-valued traits. The outcomes of the hackathon will assist my genomic analyses and predictions  (Table 1, #5, 2019-2020). \newline

\small
```{r time table,floating=FALSE,echo=FALSE,results='asis'}
b<-read.csv("Figures_related/Time_table.csv")
names(b)<-c("Tasks and Goals","2017","2018","2019","2020")

#knitr::kable(b,caption="Timetable of yearly goals")

library(pander)
panderOptions("table.style", "grid")
#panderOptions("graph.grid.lty","dashed")
pandoc.table(b,split.table=Inf,style="grid",justify="left",caption="Time table of yearly goals",emphasize.strong.cols=1,keep.line.breaks = TRUE)

#library(xtable)
#bb<-xtable(b,able.placement = "!ht")
#print(bb)
```
